src/firefly/js/visualize/HistogramOptions.jsx

```diff
@@ -75,7 +75,7 @@ var HistogramOptions = React.createClass({
                 <ValidationField
                     style={{width: 30}}
                     initialState= {{
-                        value: 0.05,
+                        value: get(histogramParams, 'falsePositiveRate')||'0.05',
```

Comment:
fyi, get takes a default value as a 3rd parameter.  
i.e.  get(histogramParams, 'falsePositiveRate', '0.05')
just sharing info.  no change is needed.  it's just as good the way you have it.